lua/golua.go

```diff
@@ -50,6 +51,8 @@ type State struct {
 
 	// User defined hook function
 	hookFn HookFunction
+
+	ctx *context.Context
```

Comment:
context.Context is already an interface, there is no reason to store it as a pointer.

Reply:
oh I didn't realize that's how that worked. Guess you learn something everyday

Reply:
fixed